scripts/ExportScripts/export_profile_monolithic.py

```diff
@@ -0,0 +1,55 @@
+from o3de.export_project import *
+from o3de import manifest
+import argparse
+import pathlib
+import os
+
+#Note: project_path and engine_path are computed before this script runs, and can be accessed via o3de_context.project_path
+#for now this script assumes an engine-centric approach, and we assume to use the source built engine for the engine_path
+
+#to use this script, go to the engine root directory, and issue the following command:
+#.\scripts\o3de.bat export-project -pp C:\path\to\project -es C:\path\to\export_profile_monolithic.py -nmp C:\path\to\build\non_mono -mp C:\path\to\build\mono -out C:\path\to\output -ll INFO
+
+parser = argparse.ArgumentParser(prog="Profile Monolithic exporter for windows",
+                                 description="Exports a profile monolithic packaged build for the project")
+parser.add_argument('-nmp', '--non-mono-build-path', type=pathlib.Path, required=True)
+parser.add_argument('-mp', '--mono-build-path',type=pathlib.Path, required=True)
+parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True)
+
+args = parser.parse_args(o3de_context.args)
+
+projects_path_cmake_option = f"-DLY_PROJECTS={str(o3de_context.project_path)}"
+
+project_json_data = manifest.get_project_json_data(project_path = o3de_context.project_path)
+project_name= project_json_data.get('project_name')
+
+
+#build non-monolithic artifacts
+process_command(['cmake', '-S', '.', '-B', args.non_mono_build_path, '-DLY_MONOLITHIC_GAME=0', projects_path_cmake_option], cwd= o3de_context.engine_path)
+
+process_command(['cmake', '--build', args.non_mono_build_path, '--target', 'Editor', '--target', 'AssetBundler', '--target', 'AssetProcessor', '--target', 'AssetProcessorBatch', '--config', 'profile'], cwd=o3de_context.engine_path)
+
+executables_dir = args.non_mono_build_path / 'bin' / 'profile'
+
+asset_processor_exe = executables_dir / 'AssetProcessorBatch.exe'
+
+asset_bundler_exe = executables_dir / 'AssetBundler.exe'
```

Comment:
Removing the `.exe` would help to allow this script to be cross-platform by allowing it to run on Linux and Mac.